src/System.ComponentModel.TypeConverter/src/System/ComponentModel/RefreshEventArgs.cs

```diff
@@ -16,7 +16,7 @@ public class RefreshEventArgs : EventArgs
         public RefreshEventArgs(object componentChanged)
         {
             ComponentChanged = componentChanged;
-            TypeChanged = componentChanged.GetType();
+            TypeChanged = componentChanged?.GetType();
```

Comment:
Why does this one use null rather than throwing an ArgumentNullException?  What's special here?

Reply:
I did this because in the constructor below we allow a null value of `typeChanged`:

```cs
        public RefreshEventArgs(Type typeChanged)
        {
            TypeChanged = typeChanged;
        }
```

Reply:
I see.  Ok, thanks.